### PR TITLE
[Update Sample] Added changes to send activity feed notifications using rsc permissions [csharp]

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by ChetanSharma-msft — APPROVED** [review]
Approving it with few comments. Please make sure to fix those before merging the PR.

File: samples/graph-rsc/csharp/README.md [hunk 1]
```diff
@@ -94,6 +95,15 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
...
+ **Tab Page**
+![tab-page](RSCDemo/Images/notify-tab.png)
+
+**Select Reciepient**
+![select-people](RSCDemo/Images/select-people.png)
+
+**Sent Notification**
+![notification](RSCDemo/Images/notification.png)
+
 ## Further Reading.
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/graph-rsc/csharp/README.md:108]: yes link is added

File: samples/graph-rsc/csharp/RSCDemo/Controllers/HomeController.cs [hunk 2]
```diff
@@ -88,17 +102,144 @@ private async Task<GraphServiceClient> GetAuthenticatedClient(string tenantId)
             return graphClient;
         }
 
+        [HttpPost]
+        [Route("GetInstalledAppList")]
+        public async Task<JsonResult> GetInstalledAppList(string reciepientUserId)
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/graph-rsc/csharp/RSCDemo/Controllers/HomeController.cs:107]: added

File: samples/graph-rsc/csharp/RSCDemo/Controllers/HomeController.cs [hunk 3]
```diff
@@ -88,17 +102,144 @@ private async Task<GraphServiceClient> GetAuthenticatedClient(string tenantId)
...
+
+        /// <summary>
+        /// Send activity feed notification to user.
+        /// </summary>
+        /// <param name="reciepientUserId"> Id of the user whom notification is to be sent</param>
+        /// <param name="appId">App id for rsc app.</param>
+        /// <returns></returns>
+        public async Task<string> SendNotification(string reciepientUserId, string appId)
+        {
+
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/graph-rsc/csharp/RSCDemo/Controllers/HomeController.cs:177]: removed

File: samples/graph-rsc/csharp/RSCDemo/Models/ResponseData.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        /// <summary>
+        /// Id of the installed app
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Teams app defination of the installed app
+        /// </summary>
+        public AppDefination TeamsAppDefinition { get; set; }
+
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/graph-rsc/csharp/RSCDemo/Models/ResponseData.cs:30]: removed

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6321510687

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving !

**Comment by Pawank-MSFT** [comment]
**Executed Build pipeline:** : https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6321510687
image

**Pull request merged**